src/sentry/conf/server.py

```diff
@@ -2412,6 +2412,20 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
             "platform": "linux/amd64",
         }
     ),
+    "taskbroker": lambda settings, options: (
+        {
+            "image": "ghcr.io/getsentry/taskbroker:latest",
+            "ports": {"50051/tcp": 50051},
+            "environment": {
+                "TASKBROKER_KAFKA_CLUSTER": "sentry_kafka:9093",
```

Comment:
I think we'll need a different host when `USE_NEW_DEVSERVICES` is active. We use

```python
host = "kafka-kafka-1" if os.environ.get("USE_NEW_DEVSERVICES") == "1" else "sentry_kafka"
```

elsewhere in devserver startup to handle the container name differences. The new-devservices container also exposes `9092` instead of `9093`.